File: angular-movieFrontend/src/Data/Request/MovieRequest.ts
export interface MovieRequest {
    title: string;
    description: string;
    genre: {
        name: string;
    };
    releaseDate: string;
    language: string;
    director: {
        name: string;
        surname: string;
    };
    duration: string;
    distributionType: string;
    status: string;
}

File: Application/Features/Medias/Movies/Common/MovieRequest.cs
using Application.Features.Directors.Common;
using Application.Features.Genres.Common;

namespace Application.Features.Medias.Movies.Common
{
    public record MovieRequest(
        string Title,
        string Description,
        GenreRequest Genre,
        DirectorRequest Director,
        DateTime ReleaseDate,
        string Language,
        TimeSpan Duration,
        string DistributionType,
        string Status
        );

}
